web-app/src/components/tournament/creator/TournamentTypeSelector.vue

```diff
@@ -5,18 +5,23 @@
       :key="item.type"
       :tournamentType="item.type"
       :iconName="item.icon"
+      :activeType="activeType"
     />
   </div>
 </template>
 
 <script lang="ts">
-import { Vue, Component } from 'vue-property-decorator';
+import { Vue, Component, Prop } from 'vue-property-decorator';
 import SelectableTournamentType from './TournamentTypeSelectorItem.vue';
+import { TournamentType } from 'app/../shared/types/Tournament';
 
 @Component({
   components: { SelectableTournamentType },
 })
 export default class TournamentTypeSelector extends Vue {
+  @Prop({ type: String || null, required: true })
+  activeType!: TournamentType | null;
```

Comment:
Is this prop nullable? I think it can be TournamenType only, when its show error try to assign one of value., then overwrite it.

Reply:
Could this prop be boolean? And tell me a different between `activeType` and `tournamentType` props

Reply:
it is null before choosing tourn type

``activeType`` is the type that is currently selected ``tournamentType`` whereas is the one u can select 
if the ``activeType`` and ``tournamentType`` of element are equal it means component should have "selected" behaviour